packages/ndla-ui/src/Article/Article.tsx

```diff
@@ -99,6 +104,9 @@ type Props = {
   children: ReactNode;
   messages: Messages;
   locale: Locale;
+  messagebox?: Boolean;
```

Comment:
Regner med at det er bedre å bruke liten b her.
```suggestion
  messagebox?: boolean;
```

Reply:
Heile denne propen er unødvendig. Så lenge messages.messageBox er truthy kan du sjekke på det på linje 200.